client/js/components/ns-autogrow.js

```diff
@@ -0,0 +1,63 @@
+/* global jQuery */
+(function($) {
+	$.fn.autogrow = function(opts) {
+		var that = $(this),
+			selector = that.selector,
+			defaults = {
```

Comment:
All these defaults aren't used (besides `context` which is used to bind events, but can it be done on `that` instead of using event delegation?)